client/src/components/InboxMessagePreview/InboxMessagePreview.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import InboxMessagePreview from "./InboxMessagePreview";
+
+import {
+  conversation1,
+  conversation2,
+  conversation3,
+  conversation4,
+  conversation5,
+  conversation6
+} from "./inboxMessagePreviewStoriesData";
+
+storiesOf("InboxMessagePreview", module)
```

Comment:
Be sure to update this so that these components go into a folder

Reply:
Added under Layout/Inbox